include/amici/steadystateproblem.h

```diff
@@ -128,17 +128,45 @@ class SteadystateProblem {
                           status) const;
 
     /**
-     * @brief Checks depending on the status of the Newton solver,
-     * solver settings, and the model, whether state sensitivities
-     * still need to be computed via a linear system solve or stored
-     * @param model pointer to the model object
-     * @param solver pointer to the solver object
-     * @param it integer with the index of the current time step
-     * @param context SteadyStateContext giving the situation for the flag
-     * @return flag telling how to process state sensitivities
+     * @brief Checks whether steadystate may be computed via Newton's method
+     * @param model model instance
+     * @param solver solver instance
+     * @return true if allowed, false otherwise
+     */
+    bool checkNewtonAllowed(const Model &model, const Solver &solver);
+    
+    /**
+     * @brief Checks whether the simulation solver or initialization provided sensitivities are correct
```

Comment:
I don't think the docstring is correct... This method actually checks whether state sensitivities have already been computed.

```suggestion
     * @brief Checks whether sensitivities have already been computed by the simulation solver 
```

Reply:
> I don't think the docstring is correct... This method actually checks whether state sensitivities have already been computed.

I don't understand what makes you think the description is not correct. This also checks whether you can use `sx` from initialization.

Reply:
My understanding of "checking whether sensitivities are correct" is a finite difference check, which compares numerically approximated sensitivities to analytically computed ones. At least this is my very first thought, although it doesn't make sense in this context. Therefore, I'd suggest another docstring.

Reply:
ah!